docs/run-a-node/setup/install.md

```diff
@@ -151,25 +151,106 @@ cd ~/node
 ```
 
 Download the updater script.
-  
+
 ```
 wget https://raw.githubusercontent.com/algorand/go-algorand-doc/master/downloads/installers/update.sh
-```  
+```
 
 + Ensure that your system knows it's an executable file.
-  
+
 ```
 chmod 544 update.sh
-```  
+```
 
 + Run the installer from within your node directory.
 
 ```
 ./update.sh -i -c stable -p ~/node -d ~/node/data -n
 ```
 
-When the installer runs, it will pull down the latest update package from S3 and install it. The `-n` option above tells the installer to not auto-start the node. If the installation succeeds the node will need to be started manually described later in this [guide](#start-node). 
+When the installer runs, it will pull down the latest update package from S3 and install it. The `-n` option above tells the installer to not auto-start the node. If the installation succeeds the node will need to be started manually described later in this [guide](#start-node).
+
+# Installing algod as a systemd service
 
+When installing using the updater script, there are several shell scripts that are bundled into the tarball that will are helpful in running `algod`. One of those is the `systemd-setup.sh` script to create a system service and the `systemd-setup-user.sh` script to create a user service.
+
+Here are the usage strings:
+
+```
+Usage: ./systemd-setup.sh username group [bindir]
+```
+
+```
+Usage: ./systemd-setup-user.sh username [bindir]
+```
+
+Note that both of them take an optional binary directory (`bindir`) parameter. This will be discussed more in the following sections.
+
+### Installing system-wide
+
+To install `algod` as a system-wide service, run the script with root privileges:
+
+```
+sudo ./systemd-setup.sh algorand algorand
+```
+
+This will create the service in `/lib/systemd/system/algorand@.service` and will have used the template `algorand@.service.template` (downloaded in the same tarball) to create the service. It includes a lot of helpful information at the top of the file and is worth perusing.
+
+The location of the binaries is needed by the template to tell `systemd` where to find `algod`. This can be controlled by the `bindir` parameter, which is the third parameter when calling the shell script.
+
```

Comment:
Clarify if this is where the bin files will be placed for systemd use or where the binary files are currently located.  Is this a directory name or the full path?  The examples imply that it is a directory name, but not clear from this description.

Reply:
Clarified that it expects an absolute path.